### ISSUE TITLE
Maybe a bug: data leak in decoder when bidirectional == True

### ISSUE THREAD

**Issue by linzeqipku** [timbmg/Sentence-VAE#22]
Model.py, Line 41:
    self.decoder_rnn = rnn(embedding_size, hidden_size, num_layers=num_layers, bidirectional=self.bidirectional, batch_first=True)

If bidirectional == True:
    When the decoder is decoding the t-th token, it can obtain information from the whole input sentence.

I think this may be a bug, and I'd appreciate that if you can help double-check this. Many thanks.

**Comment by timbmg** [comment]
Yes that is correct, so bidirectional should not be used in the decoder. Would you like to make a PR for this?
